cl8/utils/tests/test_vcard_util.py

```diff
@@ -0,0 +1,68 @@
+import pytest
+from vobject.vcard import Name
+
+from .. import vcard_util
+
+
+class TestVcardUtil:
+    def test_full_name_to_name(self):
```

Comment:
Would you mind adding a doc string giving a very short summary of what behaviour the test is checking is in place? Something like:

> check that we can free-form full name strings to structured Vcard names

or along those lines - assuming this is exercising `full_name_to_name()`

Reply:
Done in 398261fabd55a5cb670fb9c88dd1ad6a43e20a06.